library/Ivoz/Provider/Domain/Model/TerminalModel/TerminalModelRepository.php

```diff
@@ -11,4 +11,9 @@ interface TerminalModelRepository extends ObjectRepository, Selectable
      * @return TerminalModelInterface | null
      */
     public function findOneByIden(string $iden);
+
+    /**
+     * @return TerminalModelInterface | null
```

Comment:
Put this hint in php code findOneByGenericTemplate(string $genericUrlPattern): ?TerminalModelInterface